File: pset1/Mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    for (int y = 1; y <= height; y++)
    {
        int space = height;

        while (space - y > 0)
        {
            printf(" ");
            space--;
        }
        for (int x = 1; x <= y; x++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: pset6/Mario/mario.py
from cs50 import get_int

def main():

    while True:
        height = get_int("Height: ")
        if height in range(1,9):
            break
    for i in range(height):
        print(' ' * (height - 1 - i), end = '')
        print('#' * (i + 1), end = '')
        print('  ', end = '')
        print('#' * (i + 1))

main()